### ISSUE TITLE
New Csync 4 Update Slightly Breaks Celestial Tint

### ISSUE THREAD

**Issue by tapehissorchid** [sfDesat/Celestial-Tint#7]
After the new Csync 4.1.0 update that added the FixPluginTypesSerialization dependency, either that mod or something with the Csync update is causing the moving planet texture to not appear. It's just a blank color of the planet (see photo). 
![20240411225937_1](https://github.com/sfDesat/Celestial-Tint/assets/165132459/70c8fadc-b3ff-4ddb-af4d-287020cd2b92)

**Comment by sfDesat** [comment]
Try disabling LLL and only having LLLFixed enabled (if you have that installed). does this also happen on the latest version 1.1.12?

**Comment by tapehissorchid** [comment]
Yeah it looks this way regardless of if I have only LLL enabled or only LLL Fixed enabled. 

Just for context, this was happening before the v50 update today. The screenshot in the original post was taken in v49, but it is persisting after the v50 update. Today I found [this new version of LLL Fixed](https://thunderstore.io/c/lethal-company/p/Zhelderon/LethalLevelLoaderFixed/) made by someone else than [the one I have been using](https://thunderstore.io/c/lethal-company/p/Refract/LethalLevelLoaderFixed/ ). This new one is for v50. I've enabled only that and gotten most of my mods to work with it, but the blank planet issue persists.
